src/main/ui/src/components/Header/Header.jsx

```diff
@@ -15,8 +20,13 @@ export const Header = () => {
 						<SearchBar />
 						<StateDropdown />
 					</Nav>
-				</Navbar.Collapse>
+					<div style={{ display: "block", height: 49 }}>
```

Comment:
for now run all the code through jsfiddle.com in the javascript section there is a button in the top right corner of the JS part that says tidy.